pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/sink/PulsarSink.java

```diff
@@ -235,6 +239,7 @@ public void open(Map<String, Object> config, SinkContext sinkContext) throws Exc
 
     @Override
     public void write(Record<T> record) throws Exception {
+        if (record == null) return;
```

Comment:
Actually I don't think this is needed:
https://github.com/apache/pulsar/blob/master/pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java#L318

Reply:
I think the JavaInstanceRunnable is probably a better check.

Reply:
Yup that is what I mean.  We don't need the check in  sink "write"